src/api/fungible_asset.ts

```diff
@@ -36,4 +41,26 @@ export class FungibleAsset {
   }): Promise<GetFungibleAssetMetadataResponse> {
     return getFungibleAssetMetadata({ aptosConfig: this.config, ...args });
   }
+
+  /**
+   * Queries the fungible asset activities
+   *
+   * This query returns the fungible asset activities.
+   * It can be filtered by owner address, asset type, and type.
+   *
+   * @param options.pagination Where offset and limit can be specified
+   * @param options.whereCondition Where condition to filter the results
+   * @returns GetFungibleAssetActivitiesResponse A list of fungible asset metadata
+   */
+  async getFungibleAssetActivities(args: {
+    options?: {
+      pagination?: PaginationArgs;
+      whereCondition?: {
```

Comment:
i'm not a big fan of the where condition in options

Reply:
and it would be inconsistent with our other queries I feel

Reply:
agreed. also, it makes it hard to support custom user input where condition

Reply:
Updated to use bool exp